<launch>
	<param name="image_thresholder/h_lower" type="int" value="0" />
	<param name="image_thresholder/h_upper" type="int" value="12" />

	<param name="image_thresholder/s_lower" type="int" value="149" />
	<param name="image_thresholder/s_upper" type="int" value="255" />

	<param name="image_thresholder/v_lower" type="int" value="85" />
	<param name="image_thresholder/v_upper" type="int" value="255" />

	<param name="image_thresholder/gaussian" type="int" value="5" />
	
	<node name="image_thresholder" pkg="image_thresholder" type="image_thresholder_node" output="screen" />
</launch>
